src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/ReadYamlStep.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 CloudBees Inc.
```

Comment:
I can't find you in the CB directory, do you work for CloudBees? If not then you can't claim that CB owns the copyright, you or the company you work for does ;)